src/Service/DynamoDb/src/Result/BatchGetItemOutput.php

```diff
@@ -191,7 +191,7 @@ private function populateResultItemList(array $json): array
     {
         $items = [];
         foreach ($json as $item) {
-            $a = empty($item) ? [] : $this->populateResultAttributeMap($item);
+            $a = empty($item) ? null : $this->populateResultAttributeMap($item);
             if (null !== $a) {
                 $items[] = $a;
```

Comment:
This if-statement was always true before.

Reply:
maybe the fix is to remove this `if`.

All our array are initialized with `[]` on purpose. This allows users to use "array methods" or `foreach ...` even when the value is empty.